### ISSUE TITLE
Add UI to aid (guide) in the creation of custom (e.g. unrecognized remotes)

### ISSUE THREAD

**Issue by eamodio** [eamodio/vscode-gitlens#612]
Extracted from here: https://github.com/eamodio/vscode-gitlens/issues/512#issuecomment-426418740

**Comment by Megalomaniak** [comment]
Not sure if this is useful feedback(I hope so) nor if this issue is the correct place for this, but as a completely new git user I expected to be able to just right-click on remotes in gitlens side panel and see a menu item for `Add new remote`. Instead I only saw `Refresh`.

For now I've resorted to following this guide from github to add upstream remote from vsc terminal:
https://help.github.com/en/articles/configuring-a-remote-for-a-fork

It certainly works, however it does feel a bit dissatisfying having to resort to using the terminal for this IMO(what with everything else one can do through the lens). Perhaps I should open a new issue for a feature request on this?

**Comment by eamodio** [comment]
@Megalomaniak yes, please open a new feature request for that. Also PRs are very welcome 😄
